src/Interfaces/IParticlesJs.ts

```diff
@@ -0,0 +1,9 @@
+import {IOptions} from "./IOptions";
+import {Container} from "../Classes/Container";
+
+export interface IParticlesJs {
+    (tagId: string, params: IOptions): Container;
+
+    load?: (tagId: string, pathConfigJson: string, callback: (container: Container) => void) => void;
+    setOnClickHandler?: (callback: EventListenerOrEventListenerObject) => void;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5086860)